include/mapbox/geometry/feature.hpp

```diff
@@ -13,7 +13,7 @@ namespace mapbox { namespace geometry {
 
 struct value;
 
-using value_base = mapbox::util::variant<bool, int64_t, uint64_t, double, std::string,
+using value_base = mapbox::util::variant<std::nullptr_t, bool, int64_t, uint64_t, double, std::string,
```

Comment:
👍 I haven't encountered it personally but it may introduce subtle bugs, so replacing with a fake null is a good suggestion.